acceptance/lib/pxp-agent/config_helper.rb

```diff
@@ -10,17 +10,10 @@
 PCP_BROKER_PORT = 8142
 
 # SSL directories and files for our standard set of test certs
-SSL_BASE_DIR_WINDOWS = "C:\\\\cygwin64\\\\home\\\\Administrator\\\\test-resources\\\\ssl\\\\"
```

Comment:
This conflates two operations.
1. Managing getting the test certs on the system
2. Creating the config file.
I would prefer to break these apart. I have amended my PR for this concern